src/perfmon.c

```diff
@@ -1284,7 +1284,7 @@ perfmon_init_maps(void)
 #endif
 
         case ARMV7_FAMILY:
-            switch ( cpuid_info.model )
+            switch ( cpuid_info.part )
```

Comment:
The other switch does not include a case for the ARMv7 family, so this is all there is to do in this file I think.

Reply:
That's fine, the functions are fixed to the `perf_event` functions for ARM.